Add Product Team monthly meeting howto (#1311)
medic/cht-docs: content/en/contribute/medic/onboarding/team-meetings.md

---
title: "Team Meetings"
linkTitle: "Team Meetings"
weight: 4
description: >
  How the Product full-team meetings run
---

## Why
Getting to see each other all in one place at the same time, even if only online, lets people get to be around each other, learn from each other, and hear other perspectives. We're real people! ...terrific people too. This is a small space to spend some time around each other as a group. We're mostly trying to learn more about each other, celebrate recent successes, and cover the occasional big news or team-wide changes. It's mostly about being together to smile a bit.  

## When
Roughly the first Tuesday (or Wednesday, for New Zealand teammates) week of each month

## Who is there
The whole Product Team.

## Who facilitates it
Maybe you! We'll have a rotation of teammates leading the call. It doesn't mean you give a 2-hour speech at your teammates. It just means you help out with some basic logistics and guide us through the loose segments so it's not chaos. 

## How to facilitate it
### Preparations

> 1. About 2 weeks before the call, open the [Product Team Call](https://docs.google.com/document/d/14AuJ7SerLuOPESBjQlJqpBtzwSAoVf5ykTT7fjyJBT0/edit#heading=h.tpmdaeagi9ap) document and:

Reviewer: The document currently needs content. Do you think the facilitator should inform every person responsible for adding their updates?

Author: The people would probably be Gareth and me, then whoever is doing the question and the activity person. The latter two are covered, so it's just Gareth and me. I'll update the steps a bit to have that notification covered in the team-wide msg about the jamboard.

>     1. Add a new date token for "question of the week"

Reviewer: ```suggestion
    1. Add a new date token for "question of the week".
```

>     1. Randomize the list of teammate names

Reviewer: ```suggestion
    1. Randomize the list of teammate names.
```

    1. Put the name of whoever ended up first on the randomized list as the question of the week person.
    1. Notify that person via Slack that they should think up a question and add it to the doc any time prior to the call. Special note: If we have a new teammate, they give to be first and set the question. Just modify the list to make it so.

>     1. The last randomized person on the list will be in charge of "activity of the week" (anything small and fun we can all do together).

Author: I'll link a few things.

    1. Notify that person via Slack that they should find such an activity and be prepared to lead it for us all.

> 1. Create a new Jamboard in Google Drive

Author: It probably doesn't matter since it isn't intended to be long-lived, but it's easy enough to be clear about it. I'll update.

    1. link it in the agenda under "Celebrations and good fun stuff"
    1. Send a message to the whole team in Slack so they know it's available and can start putting fun things on it

### Running the call on the day-of
1. Start the call on time (mostly). We can't sit around forever, but this is mostly a fun thing, so it's ok if people are just chatting for a couple minutes while waiting for stragglers.
1. Say hello and all that
1. Do the question of the week activity
    1. Sometimes people get the order mixed up. Be prepared to nudge people if needed.
1. Move to the regular agenda. If there are special items, someone's name should be on it already.
1. Do the Jamboard

>   1. As for someone to mark the cards as you call them out. Enough people seem to take joy in this that it'll be a thing all on its own!

Reviewer: ```suggestion
  1. Ask for someone to mark the cards as you call them out. Enough people seem to take joy in this, and it'll be a thing all on its own!
```

    1. Add a new date token for "question of the week"
    1. Randomize the list of teammate names
    1. Put the name of whoever ended up first on the randomized list as the question of the week person.
    1. Notify that person via Slack that they should think up a question and add it to the doc any time prior to the call. Special note: If we have a new teammate, they give to be first and set the question. Just modify the list to make it so.
    1. The last randomized person on the list will be in charge of "activity of the week" (anything small and fun we can all do together. Past activities have included [GeoGuessr](https://www.geoguessr.com/), [Gartic Phone](https://garticphone.com/), and [Kahoot trivia](https://kahoot.com/)).
    1. Notify that person via Slack that they should find such an activity and be prepared to lead it for us all. 
1. Create a new Jamboard in the [team meeting folder](https://drive.google.com/drive/folders/1jV-8APqEQ85MOr1Irc4vRKRuKZOmI1r_)
    1. Make the Jamboard accessible for all teammates

>     1. link it in the agenda under "Celebrations and good fun stuff"

Reviewer: ```suggestion
    1. Link it in the agenda under "Celebrations and good fun stuff"
```

> 1. 1. Send a message to the whole team in Slack so they know the jamboard is available and can start putting fun things on it, and also that the agenda is open now for team-wide or org information to add.

Reviewer: ```suggestion
    1. Send a message to the whole team in Slack so they know the jamboard is available and can start putting fun things on it, and also that the agenda is open now for team-wide or org information to add.
```
